Backoffice/Manager/ContentManager.php

```diff
@@ -102,33 +104,52 @@ public function duplicateContent(ContentInterface $content, $contentId = null)
     /**
      * Duplicate a content
      *
-     * @param ContentInterface $content
-     * @param ContentInterface $lastContent
+     * @param ContentInterface $originalContent
+     * @param string           $versionName
      *
      * @return ContentInterface
      */
-    public function newVersionContent(ContentInterface $content, ContentInterface $lastContent = null)
+    public function newVersionContent(ContentInterface $originalContent, $versionName = '')
     {
-        $contentType = $this->contentTypeRepository->findOneByContentTypeIdInLastVersion($content->getContentType());
+        $lastContent = $this->contentRepository->findOneByLanguage($originalContent->getContentId(), $originalContent->getLanguage());
 
-        $lastVersion = ($contentType->isDefiningVersionable() && $lastContent !== null) ? $lastContent->getVersion() : 0;
-        $newContent = $this->cloneContent($content);
+        $lastVersion = $lastContent->getVersion();
```

Comment:
@alavieille : I think you should check that $lastContent is not null before accessing getVersion